paint/PAINT_ButtonFunctions.h

```diff
@@ -1,17 +1,17 @@
 #pragma once
-#include "WIN_Button.h"
 
 namespace win {
 	class Button;
+	class ToggleButton;
 }
 
 namespace paint {
 
 	void myAction(win::Button* button);
```

Comment:
Is `myAction` just a placeholder? If so, remove.

Reply:
done